Backpack/src/androidTest/java/net/skyscanner/backpack/nudger/BpkNudgerTest.kt

```diff
@@ -88,8 +88,10 @@ class BpkNudgerTest {
       invocationsCount++
       lastCurrent = current
     }
-    subject.value = 2
-    subject.onChangeListener = listener
+    activityRule.scenario.onActivity {
```

Comment:
i'm not sure why this suddenly started failing, but it was complaining about this done on the wrong thread